src/libraries/System.Net.NetworkInformation/ref/System.Net.NetworkInformation.cs

```diff
@@ -130,7 +130,9 @@ public abstract partial class IPGlobalProperties
         public abstract System.Net.NetworkInformation.NetBiosNodeType NodeType { get; }
         public virtual System.IAsyncResult BeginGetUnicastAddresses(System.AsyncCallback? callback, object? state) { throw null; }
         public virtual System.Net.NetworkInformation.UnicastIPAddressInformationCollection EndGetUnicastAddresses(System.IAsyncResult asyncResult) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("android")]
         public abstract System.Net.NetworkInformation.TcpConnectionInformation[] GetActiveTcpConnections();
+        [System.Runtime.Versioning.UnsupportedOSPlatform("android")]
         public abstract System.Net.IPEndPoint[] GetActiveTcpListeners();
         public abstract System.Net.IPEndPoint[] GetActiveUdpListeners();
```

Comment:
Pretty sure this one won't work either.

That raises the question: do the rest of these APIs really work on Android right now?

Reply:
Yeah, thanks, `GetActiveUdpListeners` fails with PNSE on Android as well. As for other APIs in `System.Net.NetworkInformation.IPGlobalProperties`, they all seem to fail with something like `System.UnauthorizedAccessException : UnauthorizedAccess_IODenied_Path, /proc/net/snmp\n---- System.IO.IOException : Permission denied`.

Reply:
I annotated `GetActiveUdpListeners` method too. The PR is green now and if it's enough it can be merged. Otherwise, if all other IPGlobalProperties APIs are unlikely to work then the option could be to annotation the whole class.

Reply:
Did you run the test or enabled the test assembly to check if they work?

Reply:
I ran the functional test suite using `./dotnet.sh build src/libraries/System.Net.NetworkInformation/tests/FunctionalTests /t:test /p:targetos=Android /p:targetarchitecture=x64 /p:Configuration=Release`. PNSE-throwing methods are marked as unsupported, others fail with System.UnauthorizedAccessException which I mentioned above.

Reply:
https://github.com/dotnet/runtime/issues/50567

Reply:
> others fail with System.UnauthorizedAccessException which I mentioned above.

I missed that. This is certainly not the behaviour we want. It's due to a lack of Android-specific implementation. I made #51303 to track that but won't block this PR on that.

Reply:
Thanks!